ILEditing/ILChangesLoading.cs

```diff
@@ -154,6 +154,114 @@ public override void OnModUnload()
             // (2023/07) Manually unloading IL hooks is no longer necessary as TML should automatically do it for you.
             // https://discord.com/channels/103110554649894912/445276626352209920/1099856743820959855 (links to TML server)
 
+            //Note to above: It does not do this consistently, at least for On_ hooks. Testing has proven it can cause memory leaks on mod reload. Do not trust it. Manually unload to be safe. 
```

Comment:
Can we get a source on this? There have been issues with unloading from `HookEndpointManager` in the past, but I believe they've since been resolved.